java/java.lsp.server/vscode/syntaxes/java.tmLanguage.json

```diff
@@ -40,7 +40,7 @@
 					"name": "invalid.deprecated.package_name_not_lowercase.java"
 				},
 				{
```

Comment:
`syncronized` ... a typo ?

Reply:
File is copied from `https://github.com/microsoft/vscode/blob/main/extensions/java/syntaxes/java.tmLanguage.json`